docs/software/media.md

```diff
@@ -1124,4 +1126,133 @@ A FOSS web interface media streaming server, including live TV, forked from Emby
 
 Source code: <https://github.com/jellyfin/jellyfin>
 
+## Snapcast client
+
+A FOSS multi room audio solution - this part is a client app it listens to the server and plays the audio it is sent
+
+![Snapcast logo](../assets/images/dietpi-software-media-snapcastclient.png){: width="300" height="48" loading="lazy"}
+
+The snapcast client will prompt you for the server's IP and port when installing and will be setup to start on boot up. This should work for most people but there are additional configuration parameters that can be found in the snapcast documentation: <https://github.com/badaix/snapcast#client>
+
+=== "Changing options"
+
+    It should just work but, you can change the available options in:
+
+    - `/etc/default/snapclient`
+
+    Then restart services with:
+
+    ```sh
+    systemctl restart snapclient
+    ```
+***
+
+Source code: <https://github.com/badaix/snapcast>
+
+## Snapcast server
+
+A FOSS multi room audio solution - this part is the server app it sends music to one or more clients to play.
+
+![Snapcast logo](../assets/images/dietpi-software-media-snapcastclient.png){: width="300" height="48" loading="lazy"}
+
+The snapcast serve needs to have its audio sources manually configured after installation. See the snapcast documents for more details: <https://github.com/badaix/snapcast#server>
+
+TODO: Add DietPi specific setup e.g. to connect to raspotify, airport-sync and MPD.
+
+=== "Changing options"
+
+    - `/etc/snapserver.conf`
+
+    Then restart services with:
+
+    ```sh
+    systemctl restart snapserver
+    ```
+
+=== "Using MPD as an input source"
+    
+    If you have MPD installed you can use it as an input source.
+    
+    First you need to ensure that mpd outputs to a pipe you can do this in two ways.
+    1) Install CAVA from the DietPi software list. 
+    OR
+    2) Add the following to `/etc/mpd.conf` and restart with `systemctl restart mpd`
+
+    ```sh
```

Comment:
We should not highlight shell syntax as long as it's no shell code 😉. Same below in a few cases.